docs/Configuration.md

```diff
@@ -1,60 +1,66 @@
 # Configuration
 
-The `webpack.haul.js` file is a webpack configuration used by `Haul`. The simplest config looks like this:
```

Comment:
You're markdown formatter has different config than our prettier config:
* we use single quotes instead of double quotes
* trailing spaces are added everywhere when it's compatible with ES5 (objects, arrays etc)

Reply:
I'm wondering where it's coming from because I'm using the normal prettier with eslint integration. Could you share your config?